### DISCUSSION TITLE
Github sponsors

### DISCUSSION THREAD

**Discussion by Spreeuw** [dompdf/dompdf#2329] [General]
I see that you already have a donation button at the bottom of your readme, but I think github sponsors makes it much easier to 'donate' (also on a more regular basis)
https://github.com/sponsors

By which I'm also saying: thank you SO MUCH for this awesome library! 🙏

**Comment by bsweeney** [comment]
Thanks for the kudos. I'll have to look into github sponsors a bit more. Dompdf isn't "official" enough to have its own bank account so I'd have to set it up as a personal sponsorship at present. Not sure if it's ok to use a personal bank account for an organizational github account.
